aws/resource_aws_lb_listener_rule_test.go

```diff
@@ -191,6 +191,56 @@ func TestAccAWSLBListenerRule_multipleConditionThrowsError(t *testing.T) {
 	})
 }
 
+func TestAccAWSLBListenerRule_autoPriority(t *testing.T) {
```

Comment:
I didn't mean to test static priority here since it's already tested. I was testing to switch between explicit priority and auto priority without any problem. I agree to simplify the name to `_priority` then include more error checks here.